libexec/estate-show/aws.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+if [[ -z "${ec2_id}" ]]; then
+    echo "The ec2_id for node '$name' has not been set!" >&2
+    exit 1
+fi
+if [[ -z "${aws_region}" ]]; then
+    echo "The aws_region for node '$name' has not been set!" >&2
+    exit 1
+fi
+
+aws_instance_type=$(
+aws ec2 describe-instance-attribute \
+  --attribute instanceType \
+  --query InstanceType.Value \
+  --instance-id "$ec2_id" \
+  --region "$aws_region"
+)
+
+exit_code=$?
+
+if [ ${exit_code} -eq 0 ] ; then
+  machine_type=$(case "$aws_instance_type" in
+  ('"t2.small"') echo "general-small" ;;
```

Comment:
What's the reason for only `general-small` having `echo`?

Reply:
Fixed in rebase

Reply:
https://github.com/openflighthpc/flight-action-api/pull/16/commits/7ea97b6772369b11df2ce2e789119136ef04e08f